apps/samples/Ensemble8/src/samples/java/ensemble/samples/language/swing/SampleTableModel.java

```diff
@@ -49,9 +49,9 @@
     private static ObservableList<BarChart.Series> bcData;
     private final String[] names = {"2007", "2008", "2009"};
     private Object[][] data = {
-        {new Double(567), new Double(956), new Double(1154)},
-        {new Double(1292), new Double(1665), new Double(1927)},
-        {new Double(1292), new Double(2559), new Double(2774)}
+        {Double.valueOf(567), Double.valueOf(956), Double.valueOf(1154)},
+        {Double.valueOf(1292), Double.valueOf(1665), Double.valueOf(1927)},
+        {Double.valueOf(1292), Double.valueOf(2559), Double.valueOf(2774)}
```

Comment:
Autobox?

Reply:
The array is of type Object. When LHS is Object then autobox chooses a suitable type by itself.
So here, if we autobox then the values get autoboxed to `Integer`. We do not intend to change the test behavior as part of this fix so let's keep this as `Double.valueOf`.

Reply:
I meant that we could use a double literal: `503d` or `503.0`, but if you think this is more readable then it's fine as is.

Reply:
Thanks, I would like to keep as is.